src/patternfly/components/Divider/divider.scss

```diff
@@ -1,98 +1,83 @@
-  --#{$divider}--FlexDirection: var(--#{$divider}--m-vertical--FlexDirection);
-  --#{$divider}--after--Width: var(--#{$divider}--m-vertical--after--Width);
-  --#{$divider}--after--Height: var(--#{$divider}--m-vertical--after--Height);
-
-  width: auto; // update to var breaking change
-  height: inherit; // update to var at breaking change
-
-  &::after {
-    flex-basis:
-      calc(
-        var(--#{$divider}--m-vertical--after--FlexBasis) - var(--#{$divider}--after--Inset)
-      ); // update at breaking change
-  }
+  flex-direction: column;
+  width: var(--#{$divider}--Size); // apply size to width in vertical orientation
+  height: 100%;
+}
+
+:root {
+  // * Divider
+  --#{$divider}--Display: flex;
+  --#{$divider}--Color: var(--pf-t--global--border--color--default);
+  --#{$divider}--Size: var(--pf-t--global--border--width--divider--default);
+
+  // * Divider before
+  --#{$divider}--before--FlexBasis: 100%;
 }
 
+// - Divider
 .#{$divider} {
-  // Base
-  --#{$divider}--BorderWidth--base: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$divider}--BorderColor--base: var(--#{$divider}--BackgroundColor); // update at breaking change
-  --#{$divider}--Height: var(--#{$divider}--BorderWidth--base); // remove at breaking change
-  --#{$divider}--BackgroundColor: var(--#{$pf-global}--BorderColor--100); // remove at breaking change
-  --#{$divider}--after--BackgroundColor: var(--#{$divider}--BorderColor--base); // remove at breaking change
-
-  // Borders
-  --#{$divider}--after--FlexBasis: 100%;
-  --#{$divider}--after--Inset: 0%;
-
-  // Vertical
-  --#{$divider}--m-vertical--after--FlexBasis: 100%; // remove at breaking change
-
-  // Horizontal mixin
-  --#{$divider}--m-horizontal--Display: flex;
-  --#{$divider}--m-horizontal--FlexDirection: row;
-  --#{$divider}--m-horizontal--after--Height: var(--#{$divider}--Height); // update at breaking change
-  --#{$divider}--m-horizontal--after--Width: auto;
-
-  // Vertical mixin
-  --#{$divider}--m-vertical--Display: inline-flex;
-  --#{$divider}--m-vertical--FlexDirection: column;
-  --#{$divider}--m-vertical--after--Height: auto;
-  --#{$divider}--m-vertical--after--Width: var(--#{$divider}--BorderWidth--base); // remove at breaking change
-
-  // Vertical
+  @include pf-v5-c-divider--m-horizontal; // default, set to orientation to horizontal
   @include pf-v5-hidden-visible(var(--#{$divider}--Display));
-  @include pf-v5-c-divider--m-horizontal; // set to default to horizontal
 
-  flex-direction: var(--#{$divider}--FlexDirection);
-  flex-shrink: 0;
-  align-items: center;
-  align-self: stretch;
+  display: flex;
+  align-items: stretch;
   justify-content: center;
-  border: 0; // removes the default border styling on an hr
 
-  &::after {
-    align-self: stretch;
-    justify-self: center;
-    width: var(--#{$divider}--after--Width);
-    height: var(--#{$divider}--after--Height);
+  &:is(hr) {
+    border: 0; // removes the default border styling on an hr
+  }
```

Comment:
just curious why we need to target `hr` and not just use this as a base style? Kinds of like how we apply `border: 0` to things that can be both links and buttons.